### DISCUSSION TITLE
Decrypted IPAs

### DISCUSSION THREAD

**Discussion by ErSpieltHD** [qnblackcat/uYouPlus#559] [Q&A] [answered]
Hi,

from where can I get decrypted IPAs to build it myself (GitHub Actions)? I don't own a MB.

Are appdb IPAs like https://appdb.to/app/ios/544007664 decrypted?

**Comment by arichornlover** [comment]
Hmm. I can give you these, but these decrypted ipa links will expire eventually so don’t expect these to stay.

[YouTube v17.40.5 Decrypted ipa (iOS13-Present)](https://dl.dropboxusercontent.com/s/v9a7bp3owbrcmh4/YouTube%20v17.40.5.ipa?dl=0)

[YouTube v17.39.5 Decrypted ipa (iOS13-Present)](https://dl.dropboxusercontent.com/s/830xulkyng6t3ow/YouTube%20v17.39.5.ipa?dl=0)

[YouTube v17.38.10 Decrypted ipa (iOS12-Present)](https://dl.dropboxusercontent.com/s/2sksntj67qcwn55/YouTube%20v17.38.10.ipa?dl=0)

[YouTube v17.37.3 Decrypted ipa (iOS12-Present)](https://dl.dropboxusercontent.com/s/e8nl1ai6z4hxj4y/YouTube%20v17.37.3.ipa?dl=0)

[YouTube v17.33.2 Decrypted ipa (iOS12-Present)](https://dl.dropboxusercontent.com/s/5rxwh2sfvw3bqdt/YouTube%20v17.33.2.ipa?dl=0)

[YouTube v16.16.3 Decrypted ipa (iOS12-Present)](https://dl.dropboxusercontent.com/s/ae48qrtsem2ic2x/YouTube%20v16.16.3.ipa?dl=0)

[YouTube v15.18.4 Decrypted ipa (iOS11-Present)](https://dl.dropboxusercontent.com/s/0pd1102c2xdmd0j/YouTube%20v15.18.4.ipa?dl=0)
Note: if you ever use this version I recommend Implementing the code below in Tweak.xm so you won’t be forced to update app.
```
// Hide Update Dialog: https://github.com/PoomSmart/YouTubeHeader/blob/main/YTGlobalConfig.h
%hook YTGlobalConfig
- (BOOL)shouldBlockUpgradeDialog { return YES;}
%end
```

**Reply by ErSpieltHD** [reply]
@arichorn  From where did you get them? And what does iOSxx-Present mean?

**Reply by ErSpieltHD** [reply]
Like thx for sharing but I'm still looking how to get them myself or from where to get them so I can as I've said build it myself.

Should I just ask you in the future? If yes how to contact you? You got like a discord or something?

**Reply by ErSpieltHD** [reply]
And with the tweak.xm file you mean https://github.com/qnblackcat/uYouPlus/blob/main/uYouPlus.xm right? Or in all other tweaks as well?

**Reply by arichornlover** [reply]
> @arichorn From where did you get them? And what does iOSxx-Present mean?
iOS 12 is like the app that supports that type of version and present means it’s also available for latest iOS versions

**Reply by arichornlover** [reply]
> Like thx for sharing but I'm still looking how to get them myself or from where to get them so I can as I've said build it myself.
> 
> Should I just ask you in the future? If yes how to contact you? You got like a discord or something?

https://armconverter.com/decryptedappstore/youtube here ✌️

**Reply by arichornlover** [reply]
> And with the tweak.xm file you mean https://github.com/qnblackcat/uYouPlus/blob/main/uYouPlus.xm right? Or in all other tweaks as well?

Sure, but you may have to remove a lot of tweaks for it to function properly on older versions of YouTube including settings.xm since this file is only for the new settings menu in YouTube v16.xx.x or newer

**Reply by qnblackcat** [reply]
> Like thx for sharing but I'm still looking how to get them myself or from where to get them so I can as I've said build it myself.
> 
> Should I just ask you in the future? If yes how to contact you? You got like a discord or something?

You‘ll need a jailbroken device to decrypt the app

**Reply by arichornlover** [reply]
He doesn’t need to have his device jailbroken for the site version, the site armconverter gets you the latest decrypted apps a day after their update, I did give non-jailbreaking users a better and safe alternative

**Reply by qnblackcat** [reply]
I said that he would need a jailbroken IPA to **_decrypt_** the app. Did I say something wrong? Whatever technique the site you mentioned use, it must rely on a jailbroken device. 

>... better and safe alternative

Better? Yes. Safe? No, I wouldn't say that. The safest way is to do it yourself

**Reply by arichornlover** [reply]
Oh, sorry about my inaccurate things about that message.

**Reply by yodaluca23** [reply]
@qnblackcat  Also has a GitHub with some decrypted IPAs [here](https://github.com/qnblackcat/Decrypted-IPAs)...
Speaking of which @qnblackcat I have a request... Could you please decrypt Spotify version 8.7.46?

**Reply by arichornlover** [reply]
> @qnblackcat Also has a GitHub with some decrypted IPAs [here](https://github.com/qnblackcat/Decrypted-IPAs)...

Well yeah but it’s not always consistent which is why I have my own list of decrypted ipas.
> Speaking of which @qnblackcat I have a request... Could you please decrypt Spotify version 8.7.46?

I do have a decrypted ipa of Spotify, but it isn’t the version you wanted sadly. https://www.dropbox.com/s/e5tzuuy6l0q8sty/Spotify%20v8.7.54%20Decrypted.ipa?dl=0 Spotify v8.7.54

**Comment by ErSpieltHD** [comment, marked as answer]
https://github.com/qnblackcat/Decrypted-IPAs

**Reply by arichornlover** [reply]
> https://github.com/qnblackcat/Decrypted-IPAs

I heard of that repo before, but it’s okay I’m guessing 👍
